### PR TITLE
Update dashboard for current dev branch status

### PR DESCRIPTION
**Pull request by kiephil**
Since the last updates of the dev branch a few things were broken including the possibility to open the dashboard at all. This pull request should fix these issues and also add the settings for the imageoftheday usecase.

### CODE REVIEW

File: src/dashboard/auth.test.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+beforeEach(() => {
+  // Clear all instances and calls to constructor and all methods:
+  jest.resetModules();
+});
+
+test('isAuthenticated fail', () => {
+  // Setup
+  // const localStorage = new LocalStorage('./localstorage/session');
+  // localStorage.getItem.mockReturnValue(null);
+  LocalStorage.mockImplementation(() => getNull);
```
Comment (DominikStiller) [inline src/dashboard/auth.test.ts:30]: Have you made the mocking work?

File: src/dashboard/auth.ts [hunk 2]
```diff
@@ -1,24 +1,27 @@
...
-      next();
-    } else {
-      // Not valid anymore, logging out
-      Auth.logout(sessionId);
-      res.redirect('/login');
+      return next();
     }
+    // Not valid anymore, logging out
+    Auth.logout(sessionId);
+    return res.redirect('/login');
```
Comment (DominikStiller) [inline src/dashboard/auth.ts:24]: Can you combine the upper and lower logout code for DRY purposes?

File: src/dashboard/public/js/script.js [hunk 3]
```diff
@@ -1,14 +1,42 @@
 $(document).ready(() => {
-  $('#form-unsplash').submit((e) => {
-    e.preventDefault();
+  const registerSubmitHandler = (formName) => {
+    $(`#form-${formName}`).submit((e) => {
+      e.preventDefault();
+  
+      // Serialize the form to an array
+      let formDataArray = $(`#form-${formName}`).serializeArray();
+      formDataArray.map((x) => {
```
Comment (DominikStiller) [inline src/dashboard/public/js/script.js:8]: This looks like a case for `forEach`, not `map` to me. Same thing in line 22.

File: src/dashboard/auth.test.ts (hunk 1 above)
Comment (kiephil) [reply src/dashboard/auth.test.ts:30]: No, unfortunately not

File: src/dashboard/auth.ts (hunk 2 above)
Comment (kiephil) [reply src/dashboard/auth.ts:24]: I can DRY

Comment (DominikStiller) [reply src/dashboard/auth.ts:24]: Nice one 👌

**Pull request merged**